### PR TITLE
Unify ingame message prefixes. #12923

### PR DESCRIPTION
**Pull request by rob-v**
Used "Battlefield Control" instead of "Server" or ""

### CODE REVIEW

**Review by reaperrr — APPROVED** [review]

File: OpenRA.Game/Network/UnitOrders.cs [hunk 1]
```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
+		private const string ServerChatName = "Battlefield Control";
```
Comment (abcdefg30) [inline OpenRA.Game/Network/UnitOrders.cs]: You should drop the explicit `private`. Also private variables should be `camelCase`. (See https://github.com/OpenRA/OpenRA/wiki/Coding-Standard#the-short-version . Maybe we want to make this `public` instead then?)
Also, do we want to have a `[Translate]` tag attached to this?

Comment (rob-v) [inline OpenRA.Game/Network/UnitOrders.cs]: I can either remove private or make it public and use it also in class WorldInteractionControllerWidget where "Battlefield Control" literal is used in more places.

Re Coding standard - private variables clearly camelCase, however when you search for const in OpenRA code most (all?) public and private const use CamelCase so I would keep it to be consistent.

Comment (rob-v) [inline OpenRA.Game/Network/UnitOrders.cs]: I didn't know/see [Translate] tag so far. I think it is not needed here as no chat message use [Translate].

Comment (pchote) [inline OpenRA.Game/Network/UnitOrders.cs]: Keep it here and nontranslated please.  This is not the time or way to implement translations. Removing the explicit private is sufficient.

Const variables should indeed start with a capital letter.

**Comment by pchote** [comment]
Please squash the fixup commit into the first one.

**Comment by reaperrr** [comment]
> Please squash the fixup commit into the first one.

He did.
@rob-v For future reference, it's better if you give a short "Updated" notice so every participant/subscriber to your PR is notified when you update it.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ea1184a3aae69ed90c3d34864c12dfa828f46f51...24584828e0ac14e795d71a36122c5648755b1c26)

**Pull request merged**